.github/workflows/release.yaml

```
@@ -4,6 +4,8 @@ on:
   release:
     types: 
     - created
+  schedule:
+    - cron: '0 13 * * 1-5' # 6 P.M. PST
```

```suggestion
    - cron: '0 17 * * 1-5' # 5 P.M. UTC
```

I'm a bug dummy who forgot that these times are UTC. This should go off at the top of the coming hour.